GenderPayGap.WebUI/Controllers/Admin/AdminOrganisationSectorController.cs

```diff
@@ -128,6 +183,48 @@ private void AddNewOrganisationPublicSectorType(Organisation organisation, int p
 
             organisation.LatestPublicSectorType = newOrganisationPublicSectorType;
         }
+        
+        private void UpdateAdminChangeSectorViewModelFromOrganisation(AdminChangeSectorViewModel viewModel, long organisationId)
+        {
+            viewModel.Organisation = dataRepository.Get<Organisation>(organisationId);
+
+            viewModel.InactiveUserOrganisations = dataRepository.GetAll<InactiveUserOrganisation>()
+                .Where(m => m.OrganisationId == organisationId).ToList();
+        }
+        
+        private void ValidateAdminChangeSectorViewModel(AdminChangeSectorViewModel viewModel)
+        {
+            if (!viewModel.NewSector.HasValue)
+            {
+                viewModel.AddErrorFor(m => m.NewSector, "Please select a new sector");
+            }
+
+            viewModel.ParseAndValidateParameters(Request, m => m.Reason);
+        }
+        
+        private void ChangeSector(AdminChangeSectorViewModel viewModel, long organisationId)
+        {
+            var organisation = dataRepository.Get<Organisation>(organisationId);
+
+            SectorTypes previousSector = organisation.SectorType;
+            SectorTypes newSector = viewModel.NewSector ?? SectorTypes.Unknown;

```

Comment:
Let's not allow a case where we're setting the sector to Unknown.
If NewSector is null, it's ok to throw an exception - we can even just do NewSector.Value (which will automatically throw an exception if it's null)